crates/flox-rust-sdk/src/models/environment/mod.rs

```diff
@@ -358,24 +343,38 @@ pub fn lock_manifest(
     pkgdb: &Path,
     manifest_path: &Path,
     existing_lockfile_path: Option<&Path>,
+    global_manifest_path: &Path,
 ) -> Result<serde_json::Value, EnvironmentError2> {
     let canonical_manifest_path = manifest_path
         .canonicalize()
         .map_err(EnvironmentError2::OpenManifest)?;
+
     let mut pkgdb_cmd = Command::new(pkgdb);
     pkgdb_cmd
-        .args(["manifest", "lock", "--ga-registry"])
-        .arg(canonical_manifest_path);
+        .args(["manifest", "lock"])
+        .arg("--ga-registry")
+        .arg("--global-manifest")
+        .arg(global_manifest_path);
     if let Some(lf_path) = existing_lockfile_path {
         let canonical_lockfile_path = lf_path
             .canonicalize()
             .map_err(EnvironmentError2::BadLockfilePath)?;
-        pkgdb_cmd.arg(canonical_lockfile_path);
+        pkgdb_cmd.arg("--lockfile").arg(canonical_lockfile_path);
     }
-    debug!(target: "posix", "locking manifest with command: {pkgdb_cmd:?}");
+    pkgdb_cmd.arg(canonical_manifest_path);
+
+    debug!("locking manifest with command: {pkgdb_cmd:?}");
```

Comment:
```suggestion
    debug!(target: "posix", "locking manifest with command: {pkgdb_cmd:?}");
```

for consistencies sake, can we leave them on the "posix" target for now, or remove the target from the git one as well?

Reply:
I'll just remove all of them, they end up being formatted differently for whatever reason. I've never actually seen someone use target like this. Is it meant to address some bug or edge case?

Reply:
It was meant so these dont always show up and were formatted differently to be spotted and copied more easily. There was special handling for these kinds of logs in flox-bash which we ended up trying to replicate in the rust version..